tests/sympc/tensor/replicatedshare_tensor_test.py

```diff
@@ -286,41 +286,35 @@ def test_rst_resolve_pointer(get_clients) -> None:
     assert share_pt_name == "ReplicatedSharedTensorPointer"
 
 
-@pytest.mark.parametrize("parties", [3, 5, 7])
-@pytest.mark.parametrize("security", ["malicious", "semi-honest"])
-def test_ops_public_mul_integer(get_clients, parties, security):
-    # Not encoding because truncation hasn't been implemented yet for Falcon
-    config = Config(encoder_base=1, encoder_precision=0)
+@pytest.mark.parametrize("security", ["semi-honest"])  # malicious to be added
+def test_ops_public_mul_integer(get_clients, security):
```

Comment:
I think we should still support public integer multiplication for n parties. Could you also add a test for it?

Reply:
The truncation algorithm 1 works only  for 3 three parties.

Reply:
Exactly why I am saying we could support public integer multiplication. Before truncation, I was doing it for n parties. Similar to that.

Reply:
We don't need truncation when base is 1 and precision is 0.

Reply:
Yeah , without truncation it would work, I would add it.